semanticcpg/src/main/scala/io/shiftleft/passes/linking/linker/Linker.scala

```diff
@@ -121,12 +121,12 @@ class Linker(graph: ScalaGraph) extends CpgPass(graph) {
     }
   }
 
-  private def linkToSingle[SRC_NODE_TYPE <: nodes.StoredNode](srcLabels: List[String],
```

Comment:
SRC_NODE_TYPE is only temporarily unused until @mpollmeier starts using node types again instead of JLong IDs. So please keep it.

Reply:
Not really a fan of leaving things because it will be used in some _undefined_ future. And > 2 months since the change doesn't really qualify as _temporarily_ to me.

I would agree with you if this was public interface but it's a private method that can be changed at will.

Reply:
Hmm ok.